src/ontology/mondo-edit.obo

```diff
@@ -444446,16 +444418,29 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0020772
 name: epilepsy, juvenile absence, susceptibility to, 1
+subset: gard_rare {source="GARD:2162"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:1941"}
+subset: orphanet_rare {source="Orphanet:1941"}
+subset: rare
 synonym: "EJA1" EXACT ABBREVIATION [OMIM:607631]
 synonym: "EPILEPSY, JUVENILE ABSENCE, SUSCEPTIBILITY TO, 1" EXACT [OMIM:607631]
+synonym: "epilepsy, juvenile absence, susceptibility to, type 1" EXACT [MONDORULE:1, OMIM:607631]
```

Comment:
remove "MONDORULE:1"
You added this synonym on this term based on the OMIM (MONDORULE:1 is from the early time of Mondo, and it is not the evidence that allowed you to add this synonym to this term)

Reply:
fixed